woocommerce/woocommerce-gutenberg-products-block
# Shopper → Mini Cart → Can view translated mini cart contents block

Fixes #5831, also closes #5814

### Notes

This PR adds e2e tests to verify that a shopper can see the translated Mini Cart Contents.

I used `nl_NL` instead of `fr_FR` that @alexflorisca used in #6101 because it contains more up-to-date translations at the time of writing.

~@alexflorisca I download the translation at the setup phase to make the test more reliable.~ Update: the flaky tests are caused by different issue, the translation test itself is reliable.

I also updated `shopper.block.emptyCart()` to make it language agnostic.

### Testing

#### Automated Tests

### Manual Testing

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e -- -t 'Translated Mini Cart' -i 'tests/e2e/specs/shopper/mini-cart.test.js'` to run only this test case.
5. Run `npm run test:e2e` to run the whole test suite.
